database/metadata/databases/default/tables/public_crashes_cris.yaml

```diff
@@ -97,6 +154,13 @@ insert_permissions:
         - wthr_cond_id
     comment: ""
 select_permissions:
+  - role: editor
+    permission:
+      columns:
+        - id
+        - is_temp_record
+      filter: {}
+    comment: ""
```

Comment:
is there a place in the app where editors need to read the from this table instead of `crashes`? Sorry it has been a while since I've thought through these tables, and i'm getting lost between when admins and editors permissions. 😅

edit: I see the same pattern for unit and people tables so I'm definitely missing something.

Reply:
I had to add these because when I tried soft deleting the record, I got errors until I added these permissions. Its the mutation where editors (and admins) need to read from these tables https://github.com/cityofaustin/vision-zero/blob/main/editor/queries/crash.ts#L331

Reply:
thank you for looking into this. 🙏 i will approve now that I understand better!